### PR TITLE
Fix/fixbug

### PR DESCRIPTION
**Pull request by DKnehi**

### CODE REVIEW

File: javascript-practice/src/js/controllers/userController.js [hunk 1]
```diff
@@ -24,7 +24,11 @@ export default class UserController {
         localStorage.setItem(LOCAL_STORAGE.FIRST_NAME, userData.firstName);
         localStorage.setItem(LOCAL_STORAGE.LAST_NAME, userData.lastName);
         localStorage.setItem(LOCAL_STORAGE.ROLE, userData.role);
-        this.view.redirectPage(URLS.DASHBOARD);
+        if (userData.role.toLowerCase() === 'supper admin') {
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/controllers/userController.js]: Should be constant. And typo

File: javascript-practice/src/js/helpers/validation.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ validateInputText = (text) => {
   return REGEX.TEXT.test(text)
 };
 
+
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/helpers/validation.js]: No need empty line

File: javascript-practice/src/js/constants/role.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const ROLES = {
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/constants/role.js:1]: Do we have another file for roles. If yes, consider to merge it all

File: javascript-practice/src/js/controllers/userController.js [hunk 4]
```diff
@@ -24,7 +24,11 @@ export default class UserController {
         localStorage.setItem(LOCAL_STORAGE.FIRST_NAME, userData.firstName);
         localStorage.setItem(LOCAL_STORAGE.LAST_NAME, userData.lastName);
         localStorage.setItem(LOCAL_STORAGE.ROLE, userData.role);
-        this.view.redirectPage(URLS.DASHBOARD);
+        if (userData.role.toLowerCase() === 'super admin') {
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/controllers/userController.js]: Define constant but you didn't not use it?

File: javascript-practice/src/js/templates/userTemplate.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import editIcon from '../../assets/icons/arrow-circle-down.svg';
 import deleteIcon from '../../assets/icons/arrow-circle-down-copy.svg';
-import { MONTH_NAMES, PRIMARY_ROLES } from '../constants/usersTable';
+import { MONTH_NAMES} from '../constants/month';
+import { PRIMARY_ROLES, ROLES } from '../constants/role';
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/templates/userTemplate.js]: Could you check where you use it and make sure it work?

Comment (DKnehi) [reply javascript-practice/src/js/templates/userTemplate.js]: I don't use it here, I removed it, thanks

**Pull request merged**